app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -249,6 +308,30 @@ ingress:
   # -- Affinity placement rule for the Kuma Ingress pods
   affinity: {}
 
+  # -- Security context at the pod level for control plane.
+  podSecurityContext:
+    #The values below are examples. More values can be added as needed, since the field resolves as free form.
+    #runAsNonRoot: true
+    #runAsUser: 1000
+    #runAsGroup: 3000
+    #fsGroup: 2000
+    #fsGroupChangePolicy:
+    #to support additional pod level securityContext parameters, please check:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#podsecuritycontext-v1-core 
+  
+  # -- Security context at the container level for control plane.
+  containerSecurityContext: #for overlapping securityContext between pod and container, the container's value take precedence
+    #The values below are examples. More values can be added as needed, since the field resolves as free form.
+    #allowPrivilegeEscalation: false
+    #capabilities:
+    #  drop:
+    #    - all 
+    #readOnlyRootFilesystem: true
+    #privileged: false
+    #runAsNonRoot: true
+    #runAsUser: 1000
+    #runAsGroup: 3000
+    #to support additional container level securityContext parameters, please check:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#securitycontext-v1-core
+
```

Comment:
These probably need to be removed right? As we can't add anything that will affect the sidecar in the helm chart (because it's injected). They're probably not breaking anything, they're just confusing / no-op right now.

Reply:
@johnharris85 : But these are values from the control plane testdata.